pennylane/numpy/random.py

```diff
@@ -16,18 +16,19 @@
 it works with the PennyLane :class:`~.tensor` class.
 """
 
-from autograd.numpy import random as _random
+# isort: skip_file

```

Comment:
Why was this added?

Reply:
There was a conflict between isort and CodeFactor